cli/install/register.ts

```diff
@@ -1,15 +1,43 @@
 import type { Command } from "commander"
 import { installProjectDependencies } from "lib/shared/install-project-dependencies"
+import { isGitHubUrl } from "lib/shared/is-github-url"
+import { installKicadLibrary } from "lib/shared/install-kicad-library"
+import { getPackageManager } from "lib/shared/get-package-manager"
 
 export const registerInstall = (program: Command) => {
   program
-    .command("install")
+    .command("install [package]")
     .description(
-      "Install project dependencies and generate package.json if needed",
+      "Install project dependencies, packages, or GitHub repositories (e.g., KiCad libraries)",
     )
-    .action(async () => {
+    .action(async (packageArg?: string) => {
       try {
-        await installProjectDependencies()
+        const cwd = process.cwd()
+
+        // No package specified, install all dependencies
+        if (!packageArg) {
+          await installProjectDependencies({ cwd })
+          return
+        }
+
+        // Install a specific package
+        console.log(`Installing ${packageArg}...`)
+
+        // Ensure package.json exists
+        await installProjectDependencies({ cwd })
+
+        const packageManager = getPackageManager()
+
+        // Check if it's a GitHub URL (for KiCad libraries, etc.)
+        if (!isGitHubUrl(packageArg)) {
+          // Install as a regular npm package
+          packageManager.install({ name: packageArg, cwd })
+        } else {
+          // Handle GitHub repository installation
+          await installKicadLibrary(packageArg, cwd)
```

Comment:
just because it's a github url, doesn't mean it's a kicad library, change this name to `installGithubLibrary`

Reply:
also this logic is really silly, `if (isGithubUrl(...)) { // do github url stuff }`

flip this if condition so that you're handling github installation when you check for github urls, the default behaviort is to install a package, not to assume it's a github url